config/locales/views/structures.yml

```
@@ -11,9 +11,14 @@ fr:
     structure:
       code_postal: 'Code postal : '
       rejoindre_structure: Rejoindre
+   
   admin:
+    structures:
+      show:
+        introduction: Ici vous pouvez gérer votre structure et vos collègues ayant accès à eva.<br><br>Si vous avez besoin d'aide, rendez-vous [ici](%{lien_aide})
```

Comme tu utilise le helper `md` pour rendre la traduction, tu n'es pas obligé d'écrire de l'HTML.

Tu peux faire : 
```suggestion
        introduction: |
          Ici vous pouvez gérer votre structure et vos collègues ayant accès à eva.
          
          Si vous avez besoin d'aide, rendez-vous [ici](%{lien_aide})
```